CONTRIBUTING.md

```diff
@@ -0,0 +1,466 @@
+# Contribution Guidelines
+
+If you are interested in contributing to the Substrate Documentation, please carefully read
+these contribution guidelines.
+
+We lovingly stole these guidelines from [Google's Fuchsia project](https://fuchsia.dev/), and modified them for our needs.
+
+* [Documentation Standards](#documentation-standards)
+
+* [Documentation Types](#documentation-types)
+
+* [Documentation Comments](#documentation-comments)
+
+* [Documentation Style](#documentation-style)
+
+
+## Documentation Standards
+
+A document about what to document and how to document it for people who create things that need
+documentation.
+
+### Why document?
+
+Substrate is a new framework for building blockchains. Effective documentation allows new people to join and grow the
+project by having all necessary documentation be clear and concise.
+
+### Who is the audience?
+
+The documentation described here is intended to address a technical audience, i.e. those who expect
+to implement or exercise APIs or understand the internal dynamics of the blockchain system. These
+standards are not intended for end-user product documentation.
+
+### What should I document?
+
+Document protocols, introduce essential concepts, explain how everything fits together.
+
+- Conventions: e.g. this document about documentation, code style
+
+- System Design: e.g. network stack, consensus, runtime, assumptions
+
+- APIs: e.g. JSON-RPC, runtime functions
+
+- Protocols: e.g. schemas, encodings, configuration files
+
+- Tools: e.g. `subkey`, `build-spec`, `purge-chain`
+
+- Workflows: e.g. environment set up, test methodologies, where to find various
+  parts, how to get work done
+
+### Where should I put documents?
+
+Documentation that is only for developers working on creating or maintaining
+a specific part of the code should be kept in the same directory as the source code.
+
+Substrate documentation is split into three projects:
+
+* Substrate Recipes - `substrate.dev/recipes/`
+
+   General best practice guidelines on how to develop with Substrate.
+   If you create best practice documentation about using a specific
+   feature of Substrate, you should create the documentation in the same
+   directory as the other documentation for that specific feature.
```

Comment:
```suggestion
   directory as the other documentation for that specific feature. See the [contribution guidelines](https://github.com/substrate-developer-hub/recipes/blob/master/CONTRIBUTING.md) for more details.
```

Reply:
I plan to update the CONTRIBUTING guidelines to align with this part of the fuchsia docs